test/v2/base/CommonGenerators.scala

```diff
@@ -56,4 +60,21 @@ trait CommonGenerators {
     Gen.oneOf(AuditType.values)
   }
 
+  implicit def genMessageSummary(messageId: Option[MessageId]): Arbitrary[MessageSummary] = Arbitrary {
+    for {
+      received    <- arbitrary[OffsetDateTime]
+      messageType <- Gen.oneOf(MessageType.values)
+      body        <- Gen.option(Gen.alphaNumStr)
+    } yield messageId match {
+      case Some(id) => MessageSummary(id, received, messageType, body)
+      case None =>
+        MessageSummary(
+          genShortUUID.map(MessageId(_)).sample.get,
+          received,
+          messageType,
+          body
+        )
+    }
+  }
```

Comment:
Every use case of this code uses the Some case for one (so why is it an option?) and only ever directly after generating the Id. When I suggest an arbitrary I meant like the ones above and elsewhere in the repos. i.e. `  implicit lazy val arbitraryMessageWithStateXml: Arbitrary[MessageWithStatus] =
    Arbitrary {
      for {
        messageId   <- arbitrary[MessageId]
        date        <- datesBetween(pastDate, dateNow)
        time        <- timesBetween(pastDate, dateNow)
        xml         <- Gen.const(<blankXml>message</blankXml>)
        messageType <- arbitrary[MessageType]
        status = SubmissionPending
      } yield MessageWithStatus(messageId, LocalDateTime.of(date, time), messageType, xml, status, 1)
    }`